### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by IvyChu** [IvyChu/HBHR#14]
To improve site accessibility, we need to do an audit of the current state of the application, then fix it as much as possible to enable better accessibility.

Tab ordering is the easiest to tackle and should be done first.

**Comment by IvyChu** [comment]
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
